docs/source/advanced_topics/11_memory_representation_function.rst

```diff
@@ -0,0 +1,50 @@
+Conversions Between GPU Memory Formats
+---------------------------------------------
```

Comment:
I do not see how to import the functions described in the document.

Reply:
Updated the documentation and gave examples